FixedPopupPanels/FixedPopupPanels.js

```diff
@@ -9,6 +9,7 @@
  * @exports Header
  */
 
+import classnames from 'classnames';
```

Comment:
This line should go after import statement of `getTargetByDirectionFromElement`.

Reply:
this line is deleted now.